spec.html

```diff
@@ -41765,8 +41765,8 @@ <h1>
         </dl>
         <emu-alg>
           1. Assert: The surrounding agent is in the critical section for _WL_.
-          1. Assert: _W_ is not on the list of waiters in any WaiterList.
-          1. Add _W_ to the end of the list of waiters in _WL_.
+          1. Assert: _W_ is not on the List of waiters in any WaiterList.
```

Comment:
WaiterLists actually are *not* Lists in the spec type sense, so please leave these as lowercase.